content/company/strategy/enablement/repo-management/index.md

```diff
@@ -16,127 +16,71 @@ We also have temporary custody of Authentication, with the aspiration being to s
-
-> ℹ️ The plans for this team are currently being worked out and the following summarizes the current plans.
-
 Note that the time periods are rolling time periods and the plans here are reviewed and updated monthly.
 
 ### Short term (3m)
 
 #### Monorepo support
 
 - **Current status:** In progress
-- **Expected effort:** <5% team time
+- **Expected effort:** <5% team time (our role is largely in support of the [Search team](../../code-graph/search) who owns this work)
 - **Why:** Monorepos are commonplace but exert a huge toll on systems like ours - so much so that customers feel a significant ammount of pain and with
-- Most of the previously considered work actually resides with other teams, so our role in this will largely be support.
 
-#### GitServer HA
+#### [GitServer HA](https://docs.google.com/document/d/1U5KmrVRezD1wjs1g2dBkeCJIfGTJ4dzZ8zXudJaDNNU/edit#)
 
 - **Current status:** In progress
 - **Expected effort:** ~50% team time
-- **Why:** Code host connectivity, scalability and reliability is at the heart of our product, underpinning literally all other features. As we scale the size of customers we’re trying to win and delight, we cannot avoid tactical work to expand our functional coverage (which code hosts we support and how completely), but we also need to ensure Sourcegraph can handle the current and future challenges that the largest customers will throw at it with regards to the size of their repos, number of repos and performance required.
-
-- This is partially in aid of monorepo support, and partly system resilience.
-- More information and problem context available [here](https://docs.google.com/document/d/1U5KmrVRezD1wjs1g2dBkeCJIfGTJ4dzZ8zXudJaDNNU/edit#).
+- **Why:** Code host connectivity, scalability and reliability is at the heart of our product, underpinning literally all other features. As we scale the size of customers we’re trying to win and delight, we cannot avoid tactical work to expand our functional coverage (which code hosts we support and how completely), but we also need to ensure Sourcegraph can handle the current and future challenges that the largest customers will throw at it with regards to the size of their repos, number of repos and performance required. This is partially in aid of monorepo support, and partly system resilience.
 
-1. Current work is related to replacing the hashing algorith to prevent the redistribution (rebalancing) of repos across the available Gitservers when a Gitserver is added or removed.
-2. Next step is to trial repo duplication (copies on more than one Gitserver) in Sourcegraph.com
-3. After that, making control of the replication factor customer editable (likely not in Q3)
-4. Future work (Not within Q3) - sharding monorepos across multiple Gitservers
+- [x] Current work is related to replacing the hashing algorithm to prevent the redistribution (rebalancing) of repos across the available Gitservers when a Gitserver is added or removed.
```

Comment:
```suggestion
- [ ] Current work is related to replacing the hashing algorithm to prevent the redistribution (rebalancing) of repos across the available Gitservers when a Gitserver is added or removed.
```